pilot/prompts/development/task/break_down_code_changes.prompt

```diff
@@ -9,11 +9,8 @@ Here is the current folder tree:
 ```
 
 Here are technologies that you can use:
-```
-{% for technology in technologies %}
-- {{ technology }}
-{% endfor %}
-```
+{% for tech in technologies %}
```

Comment:
This file is not actually used, but I didn't want to increase scope for this PR to include random code cleanup.